### ISSUE TITLE
SFP/USB plug collision

### ISSUE THREAD

**Issue by sbourdeauducq** [sinara-hw/Metlino#67]
![IMG_20200203_103029](https://user-images.githubusercontent.com/720864/73641540-23ab4400-46ab-11ea-9f75-fa87189fd634.jpg)

**Comment by jbqubit** [comment]
SMA J1 (FRONT_SMA_CLK_IN) is OK
SMA J2 (SMA_CLK_OUT) collides with SFP J8A

@marmeladapk Consider replacing J2 SMA on prototype hardware with short SMA/SMP connector perpendicular to the PCB.

**Comment by sbourdeauducq** [comment]
I think the SMA is fine (but double check) this issue is about the USB programming cable that definitely collides with SFP transceivers.

**Comment by gkasprow** [comment]
we cannot do much here. We can move it under SFP so it blocks only one cage. Or move the USB to the backplane side.

**Comment by jbqubit** [comment]
Looks to me like that SMA J2 also collides with SFP J8A. No?

**Comment by jbqubit** [comment]
Would a very slim micro-USB cable fit? https://www.newegg.com/p/181-02AU-000X8

**Comment by gkasprow** [comment]
Can you measure it with calipers?

**Comment by sbourdeauducq** [comment]
We can also remove one of the SFPs and replace it with e.g. a SATA connector. What is the use case for having 3?

**Comment by sbourdeauducq** [comment]
>  Or move the USB to the backplane side.

No. We do need to be able to reflash and debug Metlino in the crate in an easy manner.

**Comment by jbqubit** [comment]
Use case for three SFPs on Metlino is same as on Kasli. 

Looks like Amazon has even thinner microUSB cables. https://www.amazon.com/Ribbon-Cable-Degree-Standard-Charging/dp/B0784M2X2C
